app/assets/javascripts/dashboard/views/dash/explorative_annotations_view.js

```diff
@@ -225,10 +241,11 @@ export default class ExplorativeAnnotationsView extends React.PureComponent {
   }
 
   archiveAll = () => {
+    const selectedAnnotations = this.getFilteredTracings();
     Modal.confirm({
-      content: "Are you sure you want to archive all explorative annotations?",
+      content: `Are you sure you want to archive all ${selectedAnnotations.length} explorative annotations matching the current search query / tags.?`,
       onOk: async () => {
-        const unarchivedAnnoationIds = this.state.unarchivedTracings.map(t => t.id);
+        const unarchivedAnnoationIds = selectedAnnotations.map(t => t.id);
```

Comment:
I meant the `unarchivedAnnoationIds` variable which holds the ids of the selected annotations objects and should be named `selectedAnnotationIds` or could be directly inlined where it's used since the extraction into an own variable doesn't add much value here (it's only used once anyway).